scripts/find_duplicates.R

```diff
@@ -0,0 +1,35 @@
+library("magrittr")
+
+get_links <- function(path){
+  path %>%
+    readLines(warn = FALSE) %>%
+    .[1:which(grepl("Upcoming Events", .))] %>%
+    commonmark::markdown_xml() %>%
+    xml2::read_xml() %>%
+    xml2::xml_ns_strip() %>%
+    xml2::xml_find_all(xpath = './/link') %>%
+    xml2::xml_attr("destination")
+}
+
+get_dups <- function(rweekly_path = getwd()){
+  old_posts <- fs::dir_ls(file.path(
+    rweekly_path,"_posts"))
+  
+  old_posts <- old_posts[(length(old_posts) - 20):length(old_posts)]
+  
+  
+  old_links <- unlist(purrr::map(old_posts, get_links))
+  
+  thisweek <- get_links(file.path(rweekly_path,
+                                  "draft.md"))
+  dups <- thisweek[thisweek %in% old_links]
+  dups <- dups[dups != "http://developer.r-project.org/blosxom.cgi/R-devel/NEWS"]
```

Comment:
Could we allow for the possibility that this list might grow?

```suggestion
  acceptable_dups <- c("http://developer.r-project.org/blosxom.cgi/R-devel/NEWS")
  dups <- setdiff(dups, acceptable_dups)
```